hera_mc/geo_handling.py

```diff
@@ -16,7 +16,7 @@
 import warnings
 from astropy.time import Time, TimeDelta
 from sqlalchemy import func
-
+import matplotlib.pyplot as plt
```

Comment:
I don't understand this particular comment.

Reply:
Before, the pieces of code that imported matplotlib were all located inside the plotting functions, and so they were never triggered unless you tried to use those functions. That meant that you could `import hera_mc.geo_handling` even if you didn't have matplotlib installed. With the matplotlib import moved to the top level, you won't be able to import this module at all if you don't have matplotlib installed. Matplotlib is a pretty heavyweight dependency, and it could be genuinely difficult to install on something like a Raspberry Pi — i.e. the sort of mini-machine on which we quite plausibly will be using `hera_mc`. So I think it is better to maintain the old pattern and isolate all of the matplotlib imports to the functions that use them.